### PR TITLE
Fix wording based on Jim Young's email

### PR DESCRIPTION
**Pull request by hadrielk**
See: http://www.winpcap.org/pipermail/pcap-ng-format/2015-September/000320.html

### CODE REVIEW

File: draft-tuexen-opsawg-pcapng.xml [hunk 1]
```diff
@@ -4,12 +4,13 @@
...
 <?rfc compact='yes' ?>
 <?rfc subcompact='no' ?>
+<?rfc inline='yes' ?>
 <?rfc sortrefs='no' ?>
 <?rfc strict='yes' ?>
 
 <rfc category='info'
      ipr='trust200902'
-     docName="draft-tuexen-opsawg-pcapng-01a">
+     docName="draft-tuexen-opsawg-pcapng">
```
Comment (alagoutte) [inline draft-tuexen-opsawg-pcapng.xml]: pcapng-last ?

File: draft-tuexen-opsawg-pcapng.xml [hunk 2]
```diff
@@ -130,13 +131,31 @@
...
       <t>The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
       "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
       document are to be interpreted as described in <xref
       target='RFC2119'/>.</t>
 
+      <section title="Acronyms">
+
+        <t>The following acronyms are used throughout this document:
+
+          <list style='hanging'>
```
Comment (alagoutte) [inline draft-tuexen-opsawg-pcapng.xml]: Great idea !

**Comment by alagoutte** [comment]
Oups... comment after the merge..

Do you have check other RFC about using bytes/octets (There is a rules ?)

**Comment by hadrielk** [comment]
I don't know if there's a hard and fast rule really, but yes in my
experience IETF RFC's generally use the term "octet" for on-the-wire
encoding stuff.

-hadriel

**Comment by guyharris** [comment]
> I don't know if there's a hard and fast rule really, but yes in my
experience IETF RFC's generally use the term "octet" for on-the-wire
encoding stuff.

For what it's worth, [RFC 7322, the RFC Style Guide](http://tools.ietf.org/html/rfc7322) doesn't say anything about it, and neither does [the "Table of decisions on consistent usage in RFCs"](http://www.rfc-editor.org/rfc-style-guide/terms-online.txt).

**Pull request merged**